docs/wiki/EL/devp2p.md

```diff
@@ -1,7 +1,326 @@
-# Execution p2p
+# Execution P2P
 
 > :warning: This article is a [stub](https://en.wikipedia.org/wiki/Wikipedia:Stub), help the wiki by [contributing](/contributing.md) and expanding it.
 
-The execution layer implements [devp2p](https://github.com/ethereum/devp2p) as its communication protocol with nodes in the network. It includes various subprotocols and features: eth, snap, les, pip, wit. 
+## Peer To Peer(P2P) Communication
+Peer-to-peer (P2P) communication is a decentralized networking model where participants, known as peers, communicate directly without relying on a central server. Unlike traditional client-server architectures, where a central authority controls data flow and coordination, P2P networks distribute tasks and responsibilities among all nodes.
+
+P2P networks offer several advantages over centralized systems:
+
+- Decentralization – No single point of failure, reducing the risk of outages or censorship.
+- Scalability – The network can dynamically grow as more peers join.
+- Fault Tolerance – Nodes can leave or fail without disrupting the entire network.
+- Security & Privacy – Direct communication between peers reduces reliance on intermediaries,enhancing privacy and security.
+
+P2P communication is the foundation of many distributed applications, enabling efficient data sharing, real-time coordination, and resilient connectivity in decentralized systems.
+
+## Importance of P2P in Ethereum
```

Comment:
For the context you can add there is different p2p network in CL but then just focus it specifically on devp2p. Its importance is self explanatory..